common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt

```diff
@@ -188,7 +189,27 @@ abstract class QrScanFragment : Fragment() {
 				e.printStackTrace()
 			}
 		}, mainExecutor)
-
+		cutOut.setOnTouchListener { view: View, motionEvent: MotionEvent ->
+			when (motionEvent.action) {
+				MotionEvent.ACTION_DOWN -> true
+				MotionEvent.ACTION_UP -> {
+					// Get the MeteringPointFactory from PreviewView
+					val factory = qrCodeScanner.getMeteringPointFactory()
+
+					// Create a MeteringPoint from the tap coordinates
+					val point = factory.createPoint(motionEvent.x, motionEvent.y)
+
+					// Create a MeteringAction from the MeteringPoint, you can configure it to specify the metering mode
```

Comment:
The `you can ...` comment makes it look as if this whole code was copy pasted ;-)